### ISSUE TITLE
Once this repo ready to release, delete all of Akka-HTTP from akka/akka

### ISSUE THREAD

**Issue by ktoso** [akka/akka-http#296]
Remove all Akka HTTP classes from http://github.com/akka/akka

Do this only once:

- this repo is ready to release
- has docs 

In the meantime:

- if people PR to akka http in akka/akka, do not merge, and ask them to contribute here

**Comment by jrudolph** [comment]
Here are instructions how to migrate a PR quickly with git on the command line:

```
# add akka/akka-http remote
git remote add akka-http git@github.com:akka/akka-http.git
# fetch current state of akka-http
git fetch akka-http
# then rebase on top of akka-http (for PRs with single commit otherwise adapt `HEAD~1`)
git rebase HEAD~1 --onto akka-http/master

# add your fork of akka-http
git remote add my-akka-http git@github.com:<yourname>/akka-http.git
# push your new PR branch
git push -u my-akka-http <branch-name>

# then create PR
```

**Comment by ktoso** [comment]
Will this work properly if a commit touched streams as well?

**Comment by jrudolph** [comment]
I guess it'll conflict.
